packaging/server_config.json.template.in

```diff
@@ -1,6 +1,7 @@
 {
     "schema_name": "server_config",
     "schema_version": "v@IRODS_CONFIGURATION_SCHEMA_VERSION@",
+    "access_time_ignore_window_in_seconds": 86400,
```

Comment:
this seems high for a default, i was thinking 1 hour?

Reply:
That value comes from reading about Linux's relatime option. I'm not sure what a good default is for this. We can try 1 hour.

Reply:
Aha - very good.

And it is hard-coded in the kernel to one day.  No option to change it.

https://github.com/torvalds/linux/blob/1110ce6a1e34fe1fdc1bfe4ad52405f327d5083b/fs/inode.c#L2003-L2039

And so, we should default to one day as well.   Resolving.